application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/skill/repository/SkillLabUserProfileSyncEntity.kt

```diff
@@ -0,0 +1,20 @@
+package com.aamdigital.aambackendservice.skill.repository
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Index
+import jakarta.persistence.Table
+import java.time.OffsetDateTime
+
+@Entity
+@Table(indexes = [Index(columnList = "projectId", unique = true)])
+data class SkillLabUserProfileSyncEntity(
```

Comment:
Can we add a comment describing what exactly a sync represents? Does this mean all updates of all profiles within that project have been fetched at that time?

Reply:
Yes, added a describing.